content/docs/command-reference/push.md

```diff
@@ -188,6 +188,8 @@ Finally, we used `dvc status` to double check that all data had been uploaded.
 
 ## Example: What happens in the cache?
 
+https://youtu.be/pqnxQIORp-4
```

Comment:
In the end, the video seems to cut off while trying to show `.dvc/cache` contents.

Reply:
At 2:45, yeah.

Reply:
Fixed and waiting for youtube link in order to change it
Thanks